stdlib/public/Concurrency/Task.cpp

```diff
@@ -106,15 +106,15 @@ FutureFragment::Status AsyncTask::waitFuture(AsyncTask *waitingTask,
   auto fragment = futureFragment();
 
   auto queueHead = fragment->waitQueue.load(std::memory_order_acquire);
-  bool contextIntialized = false;
+  bool contextInitialized = false;
```

Comment:
This is a change to the code, not just to docs or comments, so someone other than me should review it.